next-tavla/app/(admin)/boards/components/SelectOrganization/index.tsx

```diff
@@ -12,8 +11,8 @@ function SelectOrganization({
     active?: TOrganization
 }) {
     return (
-        <SideNavigation className={classes.sideNav}>
-            <Heading1 className="text-rem-4">Tavler</Heading1>
+        <SideNavigation className="flex flex-col w-full md:w-1/5 g-6 bg-[var(--main-background-color)]">
```

Comment:
skal vi kanskje lage noen tailwind variabler for dette? da kunne vi sagt bg-primary (eller lignende)